VideoGenerator/library/src/main/kotlin/VideoGeneratorImplTests.kt

```diff
@@ -23,10 +39,18 @@ class VideoGeneratorImplTest {
     }
 
     @Test
-    fun testLoadFrameAndProcessFrames() {
-        videoGenerator.loadFrame(exampleImageData)
-        videoGenerator.processFrames()
+    fun testLoadFrameAndSave() {
```

Comment:
The test passes for me, but I dont see the output video in my src/resources/ folder. Any idead why not?

Reply:
Because it is automatically removed after the test. Either the line has to be commented out or we just remove that line.

Reply:
My bad, I just saw the cleanup() function